### PR TITLE
🌐 Polish translations for 3.0.0

### PR DESCRIPTION
**Pull request by DudekArtur**
Polish translation of **datamodels** and **dictionaries** added

### CODE REVIEW

**Comment by piRGoif** [comment]
Hello @DudekArtur 
Many thanks for your contribution !

What files did you used as a baseline ? 
The develop branch hosts the future 3.0.0 version. If you used current 2.7.x file, then your PR should be based on the support/2.7 branch (which hosts the upcoming 2.7.4 version)

**Comment by DudekArtur** [comment]
> Hello @DudekArtur
> Many thanks for your contribution !
> 
> What files did you used as a baseline ?
> The develop branch hosts the future 3.0.0 version. If you used current 2.7.x file, then your PR should be based on the support/2.7 branch (which hosts the upcoming 2.7.4 version)

I used the translation from my 2.7.3 instance, but I adapted some of the files to the current English version. All translations with the annotation "moved to dictionaries" were uploaded unchanged, the rest were adapted to version 3.0.0

**Comment by piRGoif** [comment]
Ok thanks ! I was asking as, as you saw, dictionaries were refactored in the 3.0.0 version...

I would recommend we keep your code as is so that no extra work is required. But that means your translations won't be available before iTop 3.0.0 beta.

The other option would be : 
* you rebasing /force push your branch on the support/2.7 branch
* adapt the files to match the ones in your iTop 2.7.3 instance
* after PR  validation + merge, we will need to add some work on the merge from support/2.7 to develop in order to keep all of your translations (for the keys that were moved in 3.0.0) - we will ask you to review this merge certainly !

Note that iTop 2.7.4 is in its final dev stage so we won't be able to integrate your work before 2.7.5.

What do you think ?

**Comment by DudekArtur** [comment]
Well. Today, in my free time, I will upload the translation to version 2.7. I use iTop and I am satisfied. Let the others be happy too.

**Comment by piRGoif** [comment]
> Well. Today, in my free time, I will upload the translation to version 2.7. I use iTop and I am satisfied. Let the others be happy too.

Wow, many thanks !!
So you can add those 2.7 translations in a new PR, based on the support/2.7 branch.

We will review/integrate your 2.7 PR, wait for the merge (support/2.7 to develop) to be done, and then we will be able to deal will this one.

File: dictionaries/pl.dictionary.itop.model.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+/**
+ * Localized data
+ *
+ * @copyright   Copyright (C) 2010-2012 Combodo SARL
+ * @license     http://opensource.org/licenses/AGPL-3.0
+ */
+
+?>
```
Comment (piRGoif) [inline dictionaries/pl.dictionary.itop.model.php:27]: This file doesn't exist anymore in EN, so you can get rid of it

File: datamodels/2.x/itop-welcome-itil/dictionaries/pl.dict.itop-welcome-itil.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ * iTop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with iTop. If not, see <http://www.gnu.org/licenses/>
+ */
+
+Dict::Add('PL PL', 'Polish', 'Polski', array(
```
Comment (piRGoif) [inline datamodels/2.x/itop-welcome-itil/dictionaries/pl.dict.itop-welcome-itil.php]: The corresponding EN file is empty, pl file should be empty to ?

File: datamodels/2.x/itop-request-mgmt/dictionaries/pl.dict.itop-request-mgmt.php [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+	'Class:UserRequest/Attribute:escalation_reason' => 'Powód - Ważny',
+	'Class:UserRequest/Attribute:escalation_reason+' => '',
+	'Class:UserRequest/Attribute:assignment_date' => 'Data przydziału',
+	'Class:UserRequest/Attribute:assignment_date+' => '',
+	'Class:UserRequest/Attribute:resolution_date' => 'Data rozwiązania',
+	'Class:UserRequest/Attribute:resolution_date+' => '',
+	'Class:UserRequest/Attribute:last_pending_date' => 'Ostatnia data trwania',
+	'Class:UserRequest/Attribute:last_pending_date+' => '',
+	'Class:UserRequest/Attribute:cumulatedpending' => 'Skumulowany czas trwania',
+	'Class:UserRequest/Attribute:cumulatedpending+' => '',
```
Comment (piRGoif) [inline datamodels/2.x/itop-request-mgmt/dictionaries/pl.dict.itop-request-mgmt.php]: Warning, both of these keys no longer exists in the develop branch, nor in this file nor anywhere else

File: datamodels/2.x/itop-request-mgmt-itil/dictionaries/pl.dict.itop-request-mgmt-itil.php [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+	'Class:UserRequest/Attribute:escalation_reason' => 'Powód - Ważny',
+	'Class:UserRequest/Attribute:escalation_reason+' => '',
+	'Class:UserRequest/Attribute:assignment_date' => 'Data przydziału',
+	'Class:UserRequest/Attribute:assignment_date+' => '',
+	'Class:UserRequest/Attribute:resolution_date' => 'Data rozwiązania',
+	'Class:UserRequest/Attribute:resolution_date+' => '',
+	'Class:UserRequest/Attribute:last_pending_date' => 'Ostatnia data trwania',
+	'Class:UserRequest/Attribute:last_pending_date+' => '',
+	'Class:UserRequest/Attribute:cumulatedpending' => 'Skumulowany czas trwania',
+	'Class:UserRequest/Attribute:cumulatedpending+' => '',
```
Comment (piRGoif) [inline datamodels/2.x/itop-request-mgmt-itil/dictionaries/pl.dict.itop-request-mgmt-itil.php]: Same remark as in itop-request-mgmt (non itil) : both of those keys were removed in the develop branch.

File: datamodels/2.x/itop-portal-base/dictionaries/pl.dict.itop-portal-base.php [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+	'Portal:Datatables:Language:Paginate:Last' => 'Ostatni',
+	'Portal:Datatables:Language:Sort:Ascending' => 'włącza sortowanie rosnąco',
+	'Portal:Datatables:Language:Sort:Descending' => 'włącza sortowanie malejąco',
+	'Portal:Autocomplete:NoResult' => 'Brak danych',
+	'Portal:Attachments:DropZone:Message' => 'Upuść pliki, aby dodać je jako załączniki',
+	'Portal:File:None' => 'Brak pliku',
+	'Portal:File:DisplayInfo' => '<a href="%2$s" class="file_download_link">%1$s</a>',
+	'Portal:File:DisplayInfo+' => '%1$s (%2$s) <a href="%3$s" class="file_open_link" target="_blank">Open</a> / <a href="%4$s" class="file_download_link">Pobierz</a>',
+	'Portal:Calendar-FirstDayOfWeek' => 'en-us', //work with moment.js locales
+	'Portal:Form:Close:Warning' => 'Chcesz opuścić ten formularz? Wprowadzone dane mogą zostać utracone',
```
Comment (piRGoif) [inline datamodels/2.x/itop-portal-base/dictionaries/pl.dict.itop-portal-base.php:71]: Check the EN file : this constant is in a dedicated section, and this other section has also the 'Portal:Form:Caselog:Entry:Close:Tooltip' key : 

```
	'Portal:Calendar-FirstDayOfWeek' => 'en-us', //work with moment.js locales
));

// Object form
Dict::Add('EN US', 'English', 'English', array(
	'Portal:Form:Caselog:Entry:Close:Tooltip' => 'Close this entry',
	'Portal:Form:Close:Warning' => 'Do you want to leave this form ? Data entered may be lost',
));
```

File: datamodels/2.x/itop-core-update/dictionaries/pl.dict.itop-core-update.php [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+	'iTopUpdate:UI:UploadMaxFileSize' => 'Wartość PHP ini upload_max_filesize: %1$s',
+
+    'iTopUpdate:UI:CanCoreUpdate:Loading' => 'Sprawdzanie plików',
+    'iTopUpdate:UI:CanCoreUpdate:Error' => 'Sprawdzanie plików nie powiodło się (%1$s)',
+    'iTopUpdate:UI:CanCoreUpdate:ErrorFileNotExist' => 'Sprawdzanie plików nie powiodło się (plik nie istnieje %1$s)',
+    'iTopUpdate:UI:CanCoreUpdate:Failed' => 'Sprawdzanie plików nie powiodło się',
+    'iTopUpdate:UI:CanCoreUpdate:Yes' => 'Aplikacja może być zaktualizowana',
+	'iTopUpdate:UI:CanCoreUpdate:No' => 'Nie można zaktualizować aplikacji: %1$s',
+	'iTopUpdate:UI:CanCoreUpdate:Warning' => 'Ostrzeżenie: aktualizacja aplikacji może się nie powieść: %1$s',
+
```
Comment (piRGoif) [inline datamodels/2.x/itop-core-update/dictionaries/pl.dict.itop-core-update.php:79]: Comparing with the EN file, missing this key : 

```suggestion
	'iTopUpdate:UI:CannotUpdateUseSetup' => 'You must use the <a href="%1$s">setup</a> to update the application.<br />Some modified files were detected, a partial update cannot be executed.~~',


```

File: datamodels/2.x/itop-attachments/dictionaries/pl.dict.itop-attachments.php [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+	'Attachment:Max_Ko' => '(Maksymalny rozmiar pliku: %1$s Ko)',
+	'Attachments:NoAttachment' => 'Brak załącznika. ',
+	'Attachments:PreviewNotAvailable' => 'Podgląd nie jest dostępny dla tego typu załącznika.',
+	'Attachments:Error:FileTooLarge' => 'Plik jest za duży do przesłania. %1$s',
+	'Attachments:Error:UploadedFileEmpty' => 'Otrzymany plik jest pusty i nie można go załączyć.
+Albo przesłałeś pusty plik,
+lub zapytaj administratora iTop, czy dysk serwera iTop jest pełny.',
+	'Attachments:Render:Icons' => 'Wyświetlaj jako ikony',
+	'Attachments:Render:Table' => 'Wyświetl jako listę',
+));
```
Comment (piRGoif) [inline datamodels/2.x/itop-attachments/dictionaries/pl.dict.itop-attachments.php:41]: Missing key here : 

```suggestion
	'UI:Attachments:DropYourFileHint' => 'Drop files anywhere in this area~~',
));

```

File: datamodels/2.x/authent-ldap/dictionaries/pl.dict.authent-ldap.php [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+//
+// Class: UserLDAP
+//
+
+Dict::Add('PL PL', 'Polish', 'Polski', array(
+	'Class:UserLDAP' => 'Użytkownik LDAP',
+	'Class:UserLDAP+' => 'Użytkownik uwierzytelniony przez LDAP',
+	'Class:UserLDAP/Attribute:password' => 'Hasło',
+	'Class:UserLDAP/Attribute:password+' => 'ciąg uwierzytelniania użytkownika',
```
Comment (piRGoif) [inline datamodels/2.x/authent-ldap/dictionaries/pl.dict.authent-ldap.php]: Those two keys aren't present anymore in the develop branch, you can remove them

```suggestion
```

**Review by piRGoif — CHANGES_REQUESTED** [review]
I'v just finish reviewing your changes, what a wonderful job you've done here !

I've just notice some missing or extra keys, see the corresponding comments.

After merging your changes I will add you to the repo README.md file in the contributors section, but if you'd like to you can also add a `@author` mention in each of the files you've created. As an example see dictionaries/de.dictionary.itop.core.php

File: dictionaries/pl.dictionary.itop.model.php (hunk 1 above)
Comment (DudekArtur) [reply dictionaries/pl.dictionary.itop.model.php:27]: Ok

File: datamodels/2.x/itop-welcome-itil/dictionaries/pl.dict.itop-welcome-itil.php (hunk 2 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-welcome-itil/dictionaries/pl.dict.itop-welcome-itil.php]: I think so. If there were no translations in the EN version, I copied the file to the PL version.

File: datamodels/2.x/authent-ldap/dictionaries/pl.dict.authent-ldap.php (hunk 8 above)
Comment (DudekArtur) [reply datamodels/2.x/authent-ldap/dictionaries/pl.dict.authent-ldap.php]: Done

File: datamodels/2.x/itop-welcome-itil/dictionaries/pl.dict.itop-welcome-itil.php (hunk 2 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-welcome-itil/dictionaries/pl.dict.itop-welcome-itil.php]: Done. PL file is empty too.

File: datamodels/2.x/itop-request-mgmt/dictionaries/pl.dict.itop-request-mgmt.php (hunk 3 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-request-mgmt/dictionaries/pl.dict.itop-request-mgmt.php]: Done

File: datamodels/2.x/itop-request-mgmt-itil/dictionaries/pl.dict.itop-request-mgmt-itil.php (hunk 4 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-request-mgmt-itil/dictionaries/pl.dict.itop-request-mgmt-itil.php]: Done

File: datamodels/2.x/itop-portal-base/dictionaries/pl.dict.itop-portal-base.php (hunk 5 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-portal-base/dictionaries/pl.dict.itop-portal-base.php:71]: Done

File: datamodels/2.x/itop-core-update/dictionaries/pl.dict.itop-core-update.php (hunk 6 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-core-update/dictionaries/pl.dict.itop-core-update.php:79]: Done

File: datamodels/2.x/itop-attachments/dictionaries/pl.dict.itop-attachments.php (hunk 7 above)
Comment (DudekArtur) [reply datamodels/2.x/itop-attachments/dictionaries/pl.dict.itop-attachments.php:41]: Done

**Comment by piRGoif** [comment]
I've just renamed the PR so that it will be easier to us :)

**Review by piRGoif — APPROVED** [review]
Thanks for the quick modifications after my first review !
I've just reviewed again, no problem. I had close the conversations, and creating this new review as approved !

**Comment by piRGoif** [comment]
Cannot merge right now, as we must first : 
1. merge #195 
2. merger support/2/7 to develop

**Comment by piRGoif** [comment]
As reviewed with the products team, #195 was rejected
In consequence I can merge this PR !

**Comment by piRGoif** [comment]
@DudekArtur I'm going to add you to the README.md's contributor list
What mention would you prefer ? Your GitHub profile name only, or more ?

**Comment by DudekArtur** [comment]
I think the GitHub profile name is enough. Regards.

**Comment by piRGoif** [comment]
Thanks, done in 8846a42 !

**Pull request merged**